package.json

```diff
@@ -75,12 +75,12 @@
     "release-it-lerna-changelog": "^1.0.3"
   },
   "engines": {
-    "node": "10.* || >= 12.*"
+    "node": ">= 12.*"
```

Comment:
This will be a major version bump I believe.  What was the specific error?  I thought v10 had another 6 days until EOL.